clients/fides-js/src/components/fides.css

```diff
@@ -737,7 +735,7 @@ div#fides-overlay-wrapper .fides-toggle .fides-toggle-display {
   display: inline-block;
   min-width: 8px;
   height: 8px;
-  margin-right: calc(0.5rem + 2px);
+  margin-right: calc(calc(var(--fides-base-font-size) * 0.5) + 2px);
```

Comment:
This originally was not tied to the variable, do we want it to be?

Reply:
Good callout. Yeah, that makes sense to me. This just controls how much space is between the down arrow icon and the text so the space there is uniform with the text spacing.